miso-web/src/main/webapp/scripts/hot_pool.js

```diff
@@ -82,6 +82,74 @@ HotTarget.pool = {
                 ids : items.map(Utils.array.getId).join(',')
               });
         }
-      }, HotUtils.printAction('pool'), ],
+      },
+      {
+        name : "Create Order",
+        action : function(pools) {
+          var platformTypes = Utils.array.deduplicateString(pools.map(function(
+              pool) {
+            return pool.platformType;
+          }));
+          if (platformTypes.length > 1) {
+            Utils
+                .showOkDialog(
+                    "Create Order",
+                    [ "All pools must be for the same platform. Currently selected:" ]
+                        .concat(platformTypes));
+          }
+          
+          var platformType = Utils.array.findFirstOrNull(function(pt) {
+            return pt.name == platformTypes[0];
+          }, Constants.platformTypes);
+          Utils.showWizardDialog('Create Order', Constants.platforms.filter(
+              function(platform) {
+                return platform.platformType == platformType.name; // TODO &&
+                // platformType.active
```

Comment:
Stray `TODO`?

Reply:
No, sadly. These need to be fixed with the container UI changes that introduce an `active`  property on platforms.

Reply:
Glorious future.